addons/sourcemod/scripting/include/utilshelper.inc

```diff
@@ -264,3 +264,19 @@ stock void StringToLowerCase(char[] input)
         input[i] = CharToLower(input[i]);
     }
 }
+
+stock int GetCurrentMapSize(bool bInMB = true)
```

Comment:
dont pass this argument and instead just return the map size in bytes and use another function to convert it in mb